templates/page.tpl.php

```diff
@@ -89,16 +89,22 @@
 </main>
 
 <footer role="contentinfo">
-  <section class="page-footer">
     <div class="wrapper">
       <section class="footer-top">
         <nav>
-          <?php print render($page['footer']); ?>
+          <?php print render($page['footer_top']); ?>
         </nav>
       </section>
+      <section>
+        <div class="footer-logo"></div>
+        <div class="footer-links">
+          <nav>
+            <?php print render($page['footer']); ?>
+          </nav>
```

Comment:
Was thinking two new settings:

1. Footer top block layout: Vertical|Horizontal
1. Footer bottom block layout: Vertical|Horizontal

Horizontal would be implemented on desktop(?) and up by default.

Thoughts?  <-- yours too @rujiali